src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,37 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+export const PostInfo = ({ post }) => {
+  const { user, title, body, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+
+        <p>
+          {' Posted by  '}
+
+          <UserInfo user={user} />
```

Comment:
user can be undefined, better to handle it
```suggestion
          {user && <UserInfo user={user} />}
```